wincrashdetect/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "44210c4a-0fe6-4702-88bf-d720e492a806",
+  "app_id": "wincrashdetect",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor your Windows hosts for system crashes.",
+    "title": "Windows Crash Detection",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Windows",
+      "Category::OS & System"
+    ]
+  },
+  "author": {
+    "support_email": "help@datadoghq.com",
+    "name": "Datadog",
+    "homepage": "https://www.datadoghq.com",
+    "sales_email": "info@datadoghq.com"
+  },
+  "oauth": {},
+  "assets": {
+    "integration": {
+      "source_type_name": "Windows Crash Detection",
+      "configuration": {},
+      "events": {
+        "creates_events": true
+      },
+      "metrics": {
+      },
```

Comment:
```suggestion
```
Can be deleted if no metrics are submitted. Will also need to delete the metadata.csv file

Reply:
I tried deleting originally; the build failed.

Reply:
I'm guessing this one:
https://github.com/DataDog/integrations-core/pull/15258/commits/ef1b086dffe4434f2f2062ea37a9edf1bf595591

That should be fixed if you delete the metadata.csv file as well.

Reply:
🤷 I can try again.